packages/core/src/transactions/TransactionRequest.ts

```diff
@@ -0,0 +1,39 @@
+export interface TransactionRequestData {
```

Comment:
Pierre, do you discussed with Brett how the generic TransactionRequestData look likes to be compatible with all libraries?

Reply:
Yes I think we could try to only add `gas` as a duplicate of `gasPrice`, and see if it works!